ann_benchmarks/runner.py

```diff
@@ -69,8 +70,14 @@ def batch_query(X):
                 pool = multiprocessing.pool.ThreadPool()
                 results = pool.map(single_query, X_test)
             else:
+                p = psutil.Process()
```

Comment:
might as well enable cpu affinity for the thread mode as well.

actually thinking about it it probably makes sense to just drop all other "modes" and only support single-threaded with cpu affinity

Reply:
I am not sure what the "thread" mode is. 
I agree that the multi-threaded mode does not make much sense (because of the overhead), but the batch mode is probably useful for testing on GPUs.

Reply:
i'm not a big fan of including batch mode since i don't think it's an apples to apples comparison. if some algorithms run in batch mode and some run in single-query mode then it's not a fair comparison.

Reply:
I agree. Probably it is best to present the results for batch mode separately.